.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
```

Comment:
Where it's being displayed? I cannot find it in your fork while creating an issue.

Reply:
I need to merge it to master there.

Reply:
See https://github.com/mdesmet/dbt-trino/issues/new/choose